webapp/sources/rudder/rudder-core/src/test/scala/com/normation/rudder/services/policies/TestBuildNodeConfiguration.scala

```diff
@@ -99,9 +99,9 @@ class TestBuildNodeConfiguration extends Specification {
   def newNode(i: Int): CoreNodeFact =
     fact1.modify(_.id).setTo(NodeId("node" + i)).modify(_.fqdn).setTo(s"node$i.localhost")
 
```

Comment:
this test wasn't testing anything, which is a bit sad since it's a nice test that exercize all the first steps of policy generation.